npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater.rb

```diff
@@ -237,6 +252,8 @@ def updated_manifest_files
         end
       end
 
+      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Note for the reviewer:

Normally I would refactor and remove these, but as we plan to swiftly follow up moving Bun to it's own ecosystem this will not be a problem.